WalletWasabi.Fluent/ViewModels/NavBar/NavBarViewModel.cs

```diff
@@ -72,15 +73,11 @@ public void Activate()
 			})
 			.Subscribe();
 
-		this.WhenAnyValue(x => x.SelectedWallet)
-			.Select(x => x?.WalletViewModel?.WalletModel)
-			.BindTo(this, x => x.SelectedWalletModel);
-
 		this.WhenAnyValue(x => x.SelectedWalletModel)
 			.Select(model => Wallets.FirstOrDefault(w => w.WalletModel == model))
 			.BindTo(this, x => x.SelectedWallet);
 
-		SelectedWallet = Wallets.FirstOrDefault(x => x.WalletModel.Name == UiContext.WalletRepository.DefaultWalletName);
+		SelectedWallet = Wallets.FirstOrDefault(x => x.WalletModel.Name == UiContext.WalletRepository.DefaultWalletName) ?? Wallets.First();
```

Comment:
This indeed fixes the issue:

```suggestion
		SelectedWallet = Wallets.FirstOrDefault(x => x.WalletModel.Name == UiContext.WalletRepository.DefaultWalletName) ?? Wallets.FirstOrDefault();
```